docs/guides/how_to_guides/configuring_data_contexts/how_to_instantiate_a_data_context_on_an_databricks_spark_cluster.rst

```diff
@@ -0,0 +1,198 @@
+.. _how_to_instantiate_a_data_context_on_a_databricks_spark_cluster:
+
+How to instantiate a Data Context on Databricks Spark cluster
+=========================================================
+
+This guide will help you instantiate a Data Context on an Databricks Spark cluster.
+
+The guide demonstrates the recommended path for instantiating a Data Context without a full configuration directory and without using the Great Expectations :ref:`command line interface (CLI) <command_line>`.
+
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
+
+    - :ref:`Followed the Getting Started tutorial and have a basic familiarity with the Great Expectations configuration<getting_started>`.
+
+Steps
+-----
+
+This how-to-guide assumes that you are using the Databricks File Store (DBFS) as the Metadata Store, and DataDocs store. The `DBFS is a file store <https://docs.databricks.com/data/databricks-file-system.html>`_ that is native to Databricks clusters and Notebooks. Files on DBFS can be written and read as if they were on a local filesystem, just by adding the `/FileStore/` prefix to the path. For information on how to configure Databricks for filesystems on Azure and AWS, please see the associated documentation in the Additional Notes section below.
+
+The following snippet shows Python code that instantiates and configures a Data Context. Copy this snippet into a cell in your Databricks Spark notebook.
```

Comment:
Maybe be more explicit - "configures a Data Context in memory, as opposed to in the great_expectations.yml"

Reply:
great point